src/utils/progressbar.js

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import {View, StyleSheet, Dimensions, Button, Text} from 'react-native';
+
+import ProgressBarAnimated from 'react-native-progress-bar-animated';
+
+const ProgressBar = () => {
+  const [progress, setProgress] = useState(70);
+
+  const barWidth = Dimensions.get('screen').width - 30;
+
+  return (
+    <View style={styles.container}>
+      <View>
+        <ProgressBarAnimated
+          width={barWidth}
+          value={progress}
+          backgroundColorOnComplete='#6CC644'
+        />
+        <View style={styles.buttonContainer}>
+          <View style={styles.buttonInner}>
+            <Button
+              title='+ 7%'
+              onPress={() => {
+                progress !== 100 ? setProgress(progress + 5) : {};
+              }}
+            />
+            <Button
+              title='- 7%'
+              onPress={() => {
+                progress !== 0 ? setProgress(progress - 5) : {};
+              }}
+            />
```

Comment:
we shouldn't have buttons to increase or decrease. these will be managed through callback from the client somewhere. like the user shouldn't specify their own progress.

Reply:
Correct. This was for demo purposes and no will no longer be needed